uber/templates/registration/new.html

```diff
@@ -117,18 +117,21 @@
 
 <br/>
 
-<table class="list">
-<tr class="header">
-    <td align="left">Name</td>
-    <td>Badge Type</td>
-    <td>Age</td>
-    <td>Cost</td>
-    <td>Paid</td>
-    {% if c.NUMBERED_BADGES %}
-        <td>Badge</td>
-    {% endif %}
-    <td></td>
-</tr>
+<table class="table datatable" data-page-length="-1">
+<thead>
+    <tr>
+        <th align="left">Name</th>
+        <th>Badge Type</th>
+        <th>Age</th>
+        <th>Cost</th>
+        <th>Paid</th>
+        {% if c.NUMBERED_BADGES %}
+            <th>Badge</th>
+        {% endif %}
+        <th></th>
+    </tr>
+</thead>
```

Comment:
The ``datatable`` library requires that we have a ``thead`` row, which is a good practice in general.  I literally didn't know that ``thead`` and ``tbody`` were even a thing for like the first ~10 years I worked on this codebase!